packages/app/package.json

```diff
@@ -77,15 +77,15 @@
     "@luckycatfactory/esbuild-graphql-loader": "3.7.0",
     "@oclif/core": "1.9.2",
     "@shopify/cli-kit": "3.25.0",
-    "@shopify/shopify-cli-extensions": "3.21.0",
+    "@shopify/ui-extensions-server-kit": "3.25.0",
```

Comment:
Since we are only using the types in development, what about making this one a `devDependency`?